src/release/issue-triaging.md

```diff
@@ -50,6 +50,10 @@ For example, `@rustbot label +T-compiler +C-bug +A-linkage +O-macos -needs-triag
 
 To see a list of all labels, check out the "labels" page next to the search bar in the issue tracker.
 
+[URLO]: https://users.rust-lang.org
+[IRLO]: https://internals.rust-lang.org/
+[Discord]: https://discord.gg/rust-lang
```

Comment:
I'm not sure how useful "official" discord is. I've left it a long time ago, because did not see reasons to be there, is it just me?

Reply:
Its help section is still decently active.

Reply:
It's mentioned in the [community page](https://www.rust-lang.org/community) of the website so AFAIK it's enough official.

I also left it many years ago but it seems there are still some ppl of the Rust project there. It's something I wish will change in the future but for now it is what it is :slightly_smiling_face: